services/movie_session.py

```diff
@@ -0,0 +1,43 @@
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: int,
+                         movie_id: int,
+                         cinema_hall_id: int) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id,
+    )
+
+
+def get_movies_sessions(session_date: int = None) -> str:
+    movie_session = MovieSession.objects.all()
+    if session_date is not None:
+        movie_session = MovieSession.objects.filter(
+            show_time__date=session_date
+        )
+    return movie_session
+
+
+def get_movie_session_by_id(movie_session_id: int) -> str:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: int = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None) -> None:
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contain string elements and can be None has typehint Optional[list[str]]